packages/common/src/common/toast.tsx

```diff
@@ -0,0 +1,217 @@
+import React from "react";
+import {
+  ToastContainer as TC,
+  toast as toastFn,
+  type Id,
+} from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import {
+  IconCheckCircleFill,
+  IconErrorFill,
+  IconInfoCircleFill,
+  Notification,
+  type NotificationType,
+} from "hds-react";
+import styled from "styled-components";
+
+type ToastProps = {
+  text: string;
+  type?: "success" | "error" | "alert" | "info";
+  label?: string;
+  ariaLabel?: string;
+  duration?: number;
+  dataTestId?: string;
+  options?: Record<string, unknown>;
+  icon?: JSX.Element;
+};
+
+const Container = styled(TC)`
+  [class^="Toastify__toast"] {
+    position: relative;
+    padding: 0;
+  }
+  --toastify-toast-bd-radius: 0;
+  --toastify-toast-width: max(30%, 320px);
+  --toastify-toast-min-height: 48px;
+  // We need to define the active color for the toast, to use it as the progress bar color...
+  --toastify-color-progress-info: rgb(0, 98, 185);
+  --toastify-color-progress-success: #007a64;
+  --toastify-color-progress-warning: #c2a251;
+  --toastify-color-progress-error: #bd2719;
```

Comment:
where do these colors come from? can we use HDS colour vars?

Reply:
Those are the values for the colors used in HDS Notification. Since the Notification component is rendered inside the ToastContainer, the ToastContainer doesn't have access to those css-variables.